File: iteration2/CSE3063F21P1_GRP16/src/main/java/Models/StudentID.java
package Models;

public class StudentID {

    private String studentEntryYear;

    private String studentEntryOrder;
    private String studentDepartment;


    public StudentID(String studentDepartment, String getStudentEntryYear, String studentEntryOrder) {

        this.studentDepartment = studentDepartment;
        this.studentEntryYear = getStudentEntryYear;
        this.studentEntryOrder = studentEntryOrder;
    }
    
    //public int getStudentID() {return 150118000;}

    public String getStudentString(){

       return String.format("1501%s%3s", studentEntryYear, studentEntryOrder).replaceAll(" ", "0");

    }


}

File: iteration3/CSE3063F21P1_GRP16/models/StudentID.py
class StudentID:
    def __init__(self, student_entry_year, student_entry_order, student_department):
        self._student_entry_year = student_entry_year
        self._student_entry_order = student_entry_order
        self._student_department = student_department

    def get_student_string(self):
        return f'1501{self._student_entry_year}{self._student_entry_order:03d}'
